src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.android.cs

```diff
@@ -5,63 +5,82 @@
 using CommunityToolkit.Maui.Core.Extensions;
 using Microsoft.Maui.Platform;
 using Activity = Android.App.Activity;
+using PlatformColor = Android.Graphics.Color;
 
 namespace CommunityToolkit.Maui.Core.Platform;
 
 [SupportedOSPlatform("Android23.0")] // StatusBar is only supported on Android 23.0+
 static partial class StatusBar
 {
-	static readonly Lazy<bool> isSupportedHolder = new(() =>
-	{
-		if (OperatingSystem.IsAndroidVersionAtLeast((int)BuildVersionCodes.M))
-		{
-			return true;
-		}
+    static readonly Lazy<bool> isSupportedHolder = new(() =>
+    {
+        if (OperatingSystem.IsAndroidVersionAtLeast((int)BuildVersionCodes.M))
+        {
+            return true;
+        }
 
-		System.Diagnostics.Trace.WriteLine($"{nameof(StatusBar)} Color + Style functionality is not supported on this version of the Android operating system. Minimum supported Android API is {BuildVersionCodes.M}");
+        System.Diagnostics.Trace.WriteLine($"{nameof(StatusBar)} Color + Style functionality is not supported on this version of the Android operating system. Minimum supported Android API is {BuildVersionCodes.M}");
 
-		return false;
-	});
+        return false;
+    });
 
-	static Activity Activity => Microsoft.Maui.ApplicationModel.Platform.CurrentActivity ?? throw new InvalidOperationException("Android Activity can't be null.");
+    static Activity Activity => Microsoft.Maui.ApplicationModel.Platform.CurrentActivity ?? throw new InvalidOperationException("Android Activity can't be null.");
 
-	static bool IsSupported => isSupportedHolder.Value;
+    static bool IsSupported => isSupportedHolder.Value;
 
-	static void PlatformSetColor(Color color)
-	{
-		if (IsSupported)
-		{
-			Activity.Window?.SetStatusBarColor(color.ToPlatform());
-		}
-	}
+    static void PlatformSetColor(Color color)
+    {
+        if (IsSupported)
```

Comment:
can we just invert this if, for better redability?

Reply:
It has been improved. Thanks @pictos for the feedback.